terraform/stages/foundation/main.tf

```diff
@@ -62,10 +85,3 @@ module "firebase" {
   firebase_init    = var.firebase_init
 }
 
-module "vpc_network" {
```

Comment:
Do we not use this module anymore?

Reply:
The VPC module is being used in GKE stage